tests/test_date_parser.py

```diff
@@ -198,3 +198,139 @@ def test_invalid_offset_timezone():
     with pytest.raises(ValueError) as excinfo:
         ParseDate("2020-01-01 12:34:56 -5")  # Not enough digits
     assert "Invalid timezone name" in str(excinfo.value)
+
+
+def test_dst_handling():
```

Comment:
I don't think this is a useful test: all the DST handling is in the pytz code, we only transform to/from it.